.github/ISSUE_TEMPLATE.md

```diff
@@ -1,9 +1,13 @@
 <!--
 Thank you for reporting an issue.
 
+*IMPORTANT* - please search our issue tracker *first* to ensure the
```

Comment:
imo the message that should reach the user when reporting a bug is that if he is confident has found unexpected behaviour of zfs should first search if someone else already reported it.

the link to how to search issues on github should be provided: https://help.github.com/articles/searching-issues/

some other issue reporting guidelines: 
media player classic homecinema https://trac.mpc-hc.org/wiki/How_to_Report_Issues
some random google result for 'how to report issues' http://aslain.com/index.php?/topic/9-how-to-report-a-bug-or-issue/
drupal: https://www.drupal.org/node/73179